apiserver/addresser/addresser_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/juju/juju/apiserver/common"
 	"github.com/juju/juju/apiserver/params"
 	apiservertesting "github.com/juju/juju/apiserver/testing"
-	"github.com/juju/juju/cmd/envcmd"
+	"github.com/juju/juju/cmd/basecmd"
```

Comment:
envcmd.SystemCommand had nothing to do with environments though. I agree with @alesstimec that `basecmd` is a more precise name.